IPython/core/oinspect.py

```diff
@@ -175,8 +175,9 @@ def object_info(
     **kw,
 ) -> InfoDict:
     """Make an object info dict with all fields present."""
-    infodict = kw
-    infodict = {k: None for k in _info_fields if k not in infodict}
+    infodict = {}
+    infodict.update(kw)
```

Comment:
Thoughts on making this a bit more succinct to make a copy of kw?
```suggestion
    infodict = dict(kw)
```

Reply:
SGTM!